.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 1
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "saturday"
+      timezone: "America/Toronto"
+
+    labels:
+      - "dependencies"
+
+    reviewers:
+      - "Seneca-CDOT/osd700-dps911-winter-2023"
+
+    open-pull-requests-limit: 5
```

Comment:
I wonder if this will be too many.  Let's try it and see how it behaves.

Reply:
Would it be ideal if I lower it down to 2 or 3?

Reply:
Try at 5 and let's see how it acts.

The reason we bumped it down in Telescope is that it was triggering a million builds and causing issues with third-party hosting providers that started to bill.  We have to keep an eye on GitHub Actions build minutes, too, so if we have too many PRs at once, it's going to cause a lot of churn since it has to be rebased etc.  Fewer might be better, but let's test.